app/views/nominations/request_nomination_invite/resend_email.html.erb

```diff
@@ -2,15 +2,19 @@
 <% content_for :before_content, govuk_back_link(text: "Back", href: :back) %>
 
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-two-thirds">
 
-        <h1 class="govuk-heading-xl">Send your school a link to use this service</h1>
+    <h1 class="govuk-heading-l">Request access to this service</h1>
```

Comment:
One minor thing - could you also set the <title> tag to the same text as this h1 please? (on line 1)